data/INRA/2016_harvest_data.ttl

```diff
@@ -0,0 +1,53 @@
+ <data/winemaking/PechRouge/Harvest/2016> a qb:DataSet; 
+  rdfs:label "Data about Harvest done on Pech Rouge in 2016"; 
+  qb:structure <DSD/harvestWeight/2016> . 
+ 
+<DSD/harvestWeight/2016> a qb:DataStructureDefinition; 
```

Comment:
There shuld not be a date here. The DSD will probably not change from year to year

Reply:
I agree with you for climatic data, it doesn't evolve. Every year, the same variables are measured. But for other data and variables, it is not the case. For example, sensorial analysis and aromas tested can be different in fonction of the year and judges.

Reply:
There is no problem in adding and removing aromas - those come from a coded list. The activity that the dataset describes remains the same and thus the structure shouldn't change. We can talk it over on the call or after